overlord/devicestate/devicestate_install_mode_test.go

```diff
@@ -139,17 +141,37 @@ func (s *deviceMgrInstallModeSuite) makeMockInstalledPcGadget(c *C) *asserts.Mod
 	return mockModel
 }
 
-func (s *deviceMgrInstallModeSuite) TestInstallModeRunChange(c *C) {
+func (s *deviceMgrInstallModeSuite) doRunChangeTestWithEncryption(c *C, grade string, tpm, bypass, res bool, e string) {
```

Comment:
maybe a small struct with tpm, bypass and res , also should 'res' be called 'encrypt' really there?  would make this a bit more readable,  also  maybe instead of passing e  just return the error at the right point and do the matching in the outer test?

Reply:
Refactored to improve readability, and also fixed a typo in a test that made a signed case be tested twice instead of testing a secured case.